build.gradle

```diff
@@ -92,7 +92,7 @@ allprojects {
         }
         mavenLocal()
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
I was getting a weird cert error from this change trying to access the new kotlin plugin in this PR, I think we might have proxy settings blocking `mavenCentral`...

Reply:
This seems to have been resolved upstream.